src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -60,6 +60,9 @@ class StringUtil
         // indices (index), appendices (appendix), prices (price)
         array('seci', 4, false, true, array('ex', 'ix', 'ice')),
 
+        // selfies (selfie)
+        array('seifles', 7, true, true, 'selfie'),
```

Comment:
I admit that it looks very weird, but in this class those words are spelled backwards. So it's "correct" to use `seifles` here.